src/components/Footer/Footer.tsx

```diff
@@ -7,14 +7,25 @@ import * as React from 'react';
 import * as Rivet from '../util/Rivet';
 import Icon from '../util/RivetIcons';
 
+interface FooterProps {
+    /**
+     * The Copyright year that appears in the footer.
+     * @see https://rivet.iu.edu/components/navigation/footer/
+     */
+    copyrightYear?: number;
```

Comment:
I thought we also wanted to accept arbitrary strings so users could specify a range?